src/main/java/core/basesyntax/Application.java

```diff
@@ -2,7 +2,23 @@
 
 public class Application {
     public static void main(String[] args) {
-        // test your code here ...
-
+        Engine engine = new Engine();
+        engine.setSystemCooling(true);
+        engine.setSystemIgnition(true);
+        engine.setSystemLubrication(true);
+        engine.setSystemStarting(true);
+        engine.setSystemSupply(true);
+        System.out.println(engine);
+        Car car = new Car();
+        car.setModel("Audi");
+        car.setColor("Green");
+        car.setCountDoors(2);
+        car.setCountWheels(4);
+        car.setEngine(engine);
+        Car carClone = car.clone();
+        car.getEngine().setSystemSupply(false);
+        System.out.println(car);
+        System.out.println(carClone);
+        System.out.println(car.equals(carClone));
```

Comment:
what are you checking by calling this last line?

Reply:
done